test/integration/short/client-pool-tests.js

```diff
@@ -559,7 +559,7 @@ describe('Client', function () {
             function wait() {
               if (waiting > 0) {
                 waiting++;
-                if (waiting > 100) {
+                if (waiting > 300) {
```

Comment:
This test was fixed on https://github.com/datastax/nodejs-driver/pull/232/commits/34532c717aecd9f3d1098cd5771329e61b419c68
You can rollback this change.